pygitguardian/models.py

```diff
@@ -794,3 +794,175 @@ class JWTService(Enum):
     """Enum for the different services GIM can generate a JWT for."""
 
     HMSL = "hmsl"
+
+
+@dataclass
+class Detector(Base, FromDictMixin):
+    name: str
+    display_name: str
+    nature: str
+    family: str
+    detector_group_name: str
+    detector_group_display_name: str
+
+
+Severity = Literal["low", "medium", "high", "critical", "unknown"]
+ValidityStatus = Literal["valid", "invalid", "failed_to_check", "no_checker", "unknown"]
+IncidentStatus = Literal["IGNORED", "TRIGGERED", "RESOLVED", "ASSIGNED"]
+Tag = Literal[
+    "DEFAULT_BRANCH",
```

Comment:
I don't see the "FALSE_POSITIVE" tag that we recently introduced.  
This is probably missing in some openAPI spec file.

Reply:
Yep unfortunately I followed the API Documentation as a start and as Aurélien warned me, it is a fair bit outdated...